content/millennium/r4/encounters/encounter.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Encounter | R4 API
+---
+
+# Encounter
+
+* TOC
+{:toc}
+
+## Overview
+
+The Encounter resource provides admissions or visits during which health care services were provided to a patient. An encounter has a class to distinguish between different health care settings such as inpatient, outpatient, emergency, etc. A patient may have one medical record number with multiple encounter numbers per facility or organization. There is substantial variance between organizations in the definition of an encounter and what events are aggregated together to constitute an encounter.
+
+The following fields are returned if valued:
+
+* [Encounter id](http://hl7.org/fhir/encounter-definitions.html#Encounter.id){:target="_blank"}
```

Comment:
link to http://hl7.org/fhir/r4/encounter-definitions.html#Encounter.id for all

Reply:
specified r4 in url for all: https://github.com/cerner/fhir.cerner.com/pull/238/commits/e3a4508a79cc3d22570a87615ace57ed020c23f1#diff-6590262e197180cc09f6ad70a2a01a5bR16